pandas_gbq/gbq.py

```diff
@@ -300,14 +315,41 @@ def load_user_account_credentials(self):
 
         return _try_credentials(self.project_id, credentials)
 
+    def get_default_credentials_path(self):
+        """
+        Gets the default path to the BigQuery credentials
+
+        .. versionadded 0.3.0
+
+        Returns
+        -------
+        Path to the BigQuery credentials
+        """
+
+        import os
+
+        if os.name == 'nt':
+            config_path = os.environ['APPDATA']
+        else:
+            config_path = os.path.join(os.path.expanduser('~'), '.config')
+
+        config_path = os.path.join(config_path, 'pandas_gbq')
```

Comment:
This is a great question! I've given this some thought and I don't feel comfortable writing default credentials into the `gcloud` path in case some users don't want to have their default `gcloud` credentials set (and I'm also worried about colliding with `gcloud`). Alternatively, users could run `gcloud auth application-default login` if they want to configure default credentials and the default `gcloud` credentials should be picked up and won't hit this code path. https://github.com/pydata/pandas-gbq/blob/master/pandas_gbq/gbq.py#L222

@tswast Thoughts?